modules/wallet/update.go

```diff
@@ -219,6 +223,10 @@ func (w *Wallet) ProcessConsensusChange(cc modules.ConsensusChange) {
 // ReceiveUpdatedUnconfirmedTransactions updates the wallet's unconfirmed
 // transaction set.
 func (w *Wallet) ReceiveUpdatedUnconfirmedTransactions(txns []types.Transaction, _ modules.ConsensusChange) {
+	if err := w.tg.Add(); err != nil {
+		panic(err)
+	}
+	defer w.tg.Done()
```

Comment:
we don't allow bare panics in our code. Use `build.Critical` (which panics in debug mode and logs otherwise) or wrap the panic in `if build.DEBUG`